libr/core/cmd_debug.c

```diff
@@ -416,7 +417,15 @@ static void cmd_debug_pid(RCore *core, const char *input) {
 			r_debug_detach (core->dbg, core->dbg->pid);
 		}
 		break;
-	case 'k':
+	case 'c': // "dpc"
+		if (core->dbg->forked_pid != -1) {
```

Comment:
i want a way to display the forked pid, instead of just being able to attach to it